### PR TITLE
exchanged img for next/Image

### PR DESCRIPTION
**Pull request by RedVeil**

### CODE REVIEW

File: packages/app/components/BatchButter/SelectToken.tsx [hunk 1]
```diff
@@ -18,14 +19,21 @@ export default function SelectToken({
   selectToken,
 }: SelectTokenProps): JSX.Element {
   const [showDropdown, setDropdown] = useState<boolean>(false);
-
+  console.log(`/images/tokens/${selectedToken.img}`)
```
Comment (anthonymartin) [inline packages/app/components/BatchButter/SelectToken.tsx:22]:
```suggestion
```

**Review by anthonymartin — CHANGES_REQUESTED** [review]
shouldn't the priority property be used for preloading images?

**Comment by RedVeil** [comment]
> shouldn't the priority property be used for preloading images?

This is still super wip. It takes a little longer than expected to swap out img with nextImages. So currently im still swapping them out and making sure everything looks as expected. After that i check priority and placeholder images

**Comment by RedVeil** [comment]
no longer applicable

**Pull request closed without merging**